test/test.js

```diff
@@ -96,6 +97,25 @@ test('plan assertions with support for promises', function (t) {
 	});
 });
 
```

Comment:
Ok, and the Promise related tests too? I'm thinking the end result would be:
- test.js
- test.Promise.js
- test.Observable.js